### ISSUE TITLE
Gitmoji ecosystem (commitlint)

### ISSUE THREAD

**Issue by jpolo** [carloscuesta/gitmoji-cli#959]
### Discussion

Hi,

I am contacting you because I use to rely on  https://github.com/arvinxx/gitmoji-commit-workflow . But the repository does seems dead and has several bugs... To me, it induce a problem of ownership/governance that I will detail below.

## Situation 
I am not using gitmoji-cli because I prefer VSCode extension, but I wanted to add some validation to ensure git message quality. that is why I used commitlint and this plugin

## Counter measures
I have considered forking the repository myself but I realized that there is a probability not to be able to maintain it, so it would not solve the problem in the end...

IMHO, this kind of repository would have a better governance if owner by the "original gitmoji team" as part of the ecosystem (changelog plugin, commitlint plugin, etc), to facilitate gitmoji adoption.
Would you consider forking it ?

Thanks in advance for your help,

### Validations

- [X] Follow our [Code of Conduct](https://github.com/carloscuesta/gitmoji-cli/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/carloscuesta/gitmoji-cli/blob/master/.github/CONTRIBUTING.md).
- [X] Check that there isn't already an issue requesting the same feature.

**Comment by carloscuesta** [comment]
Hey! 

I think that's the beauty of Open Source! We never had an "original gitmoji" team as you say, the reality is that everyone dedicates free time to Open Source and we never built any governance around the project, aside from me which I'm maintaining `gitmoji` and `gitmoji-cli` the community built different projects based on their needs which is awesome ❤️!

Having governance also implies more bureaucracy and processes and can be tricky to manage as well!

Regarding your original question, I have no plans or interest to fork the project you mentioned since I don't have a lot of time right now to put into new projects. 

If you think you can improve it, I would suggest contacting the original author so you can collaborate with them in order to fix the issues you're mentioning before trying to fork the project. In my experience from a maintainer point of view this would best way to help with a project you plan to use.

I will try to think a bit more around the governance and having different projects under a "gitmoji organization" to see if it would make sense to build something like that.

Thanks for opening the issue!

Have a great day 😊

**Comment by jpolo** [comment]
I have already contacted the maintainer of the repository, but there are no answer and there is already a pile of opened issues... I even proposed to submit a PR, that would correct the issues. That is why I proposed this change of governance.

I can totally understand the reasons why you cannot fork the repository at the moment and I thank you for taking the time to provide an answer.

In the future if you decide to create an organization to group the gitmoji ecosystem, (and therefore include the commitlint repo in some way), I would be glad to help to solve the pending issues.

Nice day to you too!
